Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM quay.io/deis/go-dev:0.17.0
+# This Dockerfile is used to bundle the source and all dependencies into an image for testing.
+
+RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-jessie main" \
+  | tee /etc/apt/sources.list.d/google-cloud-sdk.list \
+  && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg \
+  | apt-key add - \
+  && apt-get update \
+  && apt-get install -y google-cloud-sdk \
+  --no-install-recommends \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY glide.yaml /go/src/github.com/deis/controller-sdk-go/
```

Comment:
The reason is I copy the `glide.yaml` first so that we only run `glide install` if the `glide.yaml` changes. Otherwise any file change would bust the `glide install` cache.